ios/FluentUI/Button/Button.swift

```diff
@@ -221,6 +243,12 @@ open class Button: UIButton {
 
         titleLabel?.font = style.titleFont
```

Comment:
similar to our pillbutton fix we probably need to update font for ios 15?

Reply:
Do you mean the bug with the segmented control? If we had the FAB on this branch we would, but since these buttons are meant to scale this should be okay.